testcase/bank2/cmd/main.go

```diff
@@ -33,17 +33,18 @@ import (
 )
 
 var (
-	accounts    = flag.Int("accounts", 1000000, "the number of accounts")
-	interval    = flag.Duration("interval", 2*time.Second, "the interval")
-	concurrency = flag.Int("concurrency", 200, "concurrency worker count")
-	retryLimit  = flag.Int("retry-limit", 200, "retry count")
-	longTxn     = flag.Bool("long-txn", true, "enable long-term transactions")
-	contention  = flag.String("contention", "low", "contention level, support values: high / low, default value: low")
-	pessimistic = flag.Bool("pessimistic", false, "use pessimistic transaction")
-	minLength   = flag.Int("min-value-length", 0, "minimum value inserted into rocksdb")
-	maxLength   = flag.Int("max-value-length", 128, "maximum value inserted into rocksdb")
-	replicaRead = flag.String("tidb-replica-read", "leader", "tidb_replica_read mode, support values: leader / follower / leader-and-follower, default value: leader.")
-	dbname      = flag.String("dbname", "test", "name of database to test")
+	accounts            = flag.Int("accounts", 1000000, "the number of accounts")
+	interval            = flag.Duration("interval", 2*time.Second, "the interval")
+	concurrency         = flag.Int("concurrency", 200, "concurrency worker count")
+	retryLimit          = flag.Int("retry-limit", 200, "retry count")
+	longTxn             = flag.Bool("long-txn", true, "enable long-term transactions")
+	contention          = flag.String("contention", "low", "contention level, support values: high / low, default value: low")
+	pessimistic         = flag.Bool("pessimistic", false, "use pessimistic transaction")
+	minLength           = flag.Int("min-value-length", 0, "minimum value inserted into rocksdb")
+	maxLength           = flag.Int("max-value-length", 128, "maximum value inserted into rocksdb")
+	replicaRead         = flag.String("tidb-replica-read", "leader", "tidb_replica_read mode, support values: leader / follower / leader-and-follower, default value: leader.")
+	dbname              = flag.String("dbname", "test", "name of database to test")
+	randomToggleFeature = flag.Bool("random-toggle-feature", false, "")
```

Comment:
I think it's a little vague to me since there only two toggle-feature here(tidb_enable_async_commit and enable-compaction-filter). Rest LGTM

Reply:
I mean that `random-toggle-feature` maybe not a good name, but I have no idea about the name.